### PR TITLE
Support for Certificate Pinning in Omise Swift

### PR DESCRIPTION
**Pull request by pitiphong-p**

### CODE REVIEW

File: OmiseSwift/Client.swift [hunk 1]
```diff
@@ -44,3 +44,49 @@ open class Client: NSObject {
...
+                creditial = nil
+                challengeDisposition = .cancelAuthenticationChallenge
+                return
+        }
+        var secresult = SecTrustResultType.invalid
+        let status = SecTrustEvaluate(serverTrust, &secresult)
+        
+        guard errSecSuccess == status,
+            let serverCertificate = SecTrustGetCertificateAtIndex(serverTrust, 0)else {
+                creditial = nil
```
Comment (chakrit) [inline OmiseSwift/Client.swift]: `credential`

File: OmiseSwift/Config.swift [hunk 2]
```diff
@@ -6,7 +6,9 @@ open class Config: NSObject {
     open let secretKey: String?
     
     open let callbackQueue: OperationQueue
-    
+
+    public let pinningSignature: Data?
```
Comment (chakrit) [inline OmiseSwift/Config.swift]: I think we should just hide this completely, hard-code the certs.

**Review by chakrit — CHANGES_REQUESTED** [review]

File: OmiseSwift/Config.swift (hunk 2 above)
Comment (pitiphong-p) [reply OmiseSwift/Config.swift]: I hide this signature in the Client now

File: OmiseSwift/Client.swift (hunk 1 above)
Comment (pitiphong-p) [reply OmiseSwift/Client.swift]: Thank you

**Review by chakrit — APPROVED** [review]
👍

**Pull request merged**